### PR TITLE
Information Exposure

### PR DESCRIPTION
**Pull request by johniak**
[AB#207787](https://unicef.visualstudio.com/4e044e8d-bc28-4768-8074-f80ff6e4a0e5/_workitems/edit/207787) Information Exposure

### CODE REVIEW

**Review by patryk-dabrowski — APPROVED** [review]

File: backend/hct_mis_api/apps/household/views.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def get(self, request: Request) -> Response:
 
         try:
             data = get_household_or_individual(tax_id, registration_id, business_area_code)
-        except Exception as exception:
-            return Response({"status": "not found", "error_message": str(exception)}, status=404)
+        except Exception:  # pragma: no cover
+            return Response({"status": "not found", "error_message": "Household not Found"}, status=404)
```
Comment (domdinicola) [inline backend/hct_mis_api/apps/household/views.py:116]: can we capture the exception in sentry?

Comment (johniak) [reply backend/hct_mis_api/apps/household/views.py:119]: I added logging, but changing exception to something more specific is not possible, without wrewriting this functions
image

Comment (domdinicola) [reply backend/hct_mis_api/apps/household/views.py:119]: I see Imported Document here... I'm a little scared!

**Comment by domdinicola** [comment]
@johniak can we finish this?

**Comment by domdinicola** [comment]
@pavlo-mk can you adjust the migrations?

**Comment by domdinicola** [comment]
@pavlo-mk can you fix the tests?

**Comment by domdinicola** [comment]
@pavlo-mk test fails again

**Review by pkujawa — APPROVED** [review]

**Pull request merged**